config/RSK.mainnet.json

```diff
@@ -3,7 +3,11 @@
     "node": {
         "network": "mainnet",
         "address": "https://rsk-graph1.sovryn.app/rpc",
-        "log_level": "info"
+        "log_level": "info",
+        "graph_graphql_query_timeout": 10,
+        "graph_sql_statement_timeout": 10,
+        "graph_graphql_max_depth": 16,
```

Comment:
max depth for mainnet should be 4
this will block the attempt to bring down the subgraph but also will block the graphql scheme with all the documentation to come up

Reply:
This is resolved by max_complexity change